Scrum-events-csm-guide.html

```
@@ -0,0 +1,1047 @@
+          <li><span class="ref-name">Scrum Alliance, Certified ScrumMaster (CSM) Certification</span><a class="ref-url" href="https://www.scrumalliance.org/get-certified/scrum-master-track/certified-scrummaster" target="_blank" rel="noopener">https://www.scrumalliance.org/get-certified/scrum-master-track/certified-scrummaster</a></li>
+          <li><span class="ref-name">Scrum Alliance, CSM Learning Objectives</span><a class="ref-url" href="https://www.scrumalliance.org/media/certifications/los/csm_learning_objectives_2022.pdf" target="_blank" rel="noopener">https://www.scrumalliance.org/media/certifications/los/csm_learning_objectives_2022.pdf</a></li>
+          <li><span class="ref-name">Scrum Alliance, Scrum Foundations Learning Objectives</span><a class="ref-url" href="https://www.scrumalliance.org/media/certifications/los/scrum_foundations_learning_objectives_2022.pdf" target="_blank" rel="noopener">https://www.scrumalliance.org/media/certifications/los/scrum_foundations_learning_objectives_2022.pdf</a></li>
+          <li><span class="ref-name">Scrum Alliance, Scrum Values</span><a class="ref-url" href="https://www.scrumalliance.org/about-scrum/values" target="_blank" rel="noopener">https://www.scrumalliance.org/about-scrum/values</a></li>
+          <li><span class="ref-name">Scrum Alliance Support, How do I take the CSM test?</span><a class="ref-url" href="https://support.scrumalliance.org/hc/en-us/articles/360002112772-How-do-I-take-the-CSM-test" target="_blank" rel="noopener">https://support.scrumalliance.org/hc/en-us/articles/360002112772-How-do-I-take-the-CSM-test</a></li>
+          <li><span class="ref-name">Manifesto for Agile Software Development</span><a class="ref-url" href="https://agilemanifesto.org" target="_blank" rel="noopener">https://agilemanifesto.org</a></li>
+        </ul>
+      </div>
+    </section>
+
+    <footer>
+      Scrum, Sprint, Sprint Planning, Daily Scrum, Sprint Review, Sprint Retrospective はScrum Guideで定義される用語です。Certified ScrumMaster(R) および CSM(R) はScrum Alliance(R)の登録商標です。本ページはScrum Allianceによる公式認定教材ではありません。 The Scrum Guide is &copy; 2020 Ken Schwaber and Jeff Sutherland, and is made available under the Creative Commons Attribution-ShareAlike 4.0 International License(CC BY-SA 4.0) &mdash; <a href=\"https://creativecommons.org/licenses/by-sa/4.0/legalcode\" target=\"_blank\" rel=\"noopener\">creativecommons.org/licenses/by-sa/4.0/legalcode</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)